numpy/core/src/umath/ufunc_object.c

```diff
@@ -1205,6 +1216,10 @@ get_ufunc_arguments(PyUFuncObject *ufunc,
     *out_extobj = NULL;
     Py_XDECREF(*out_typetup);
     *out_typetup = NULL;
+    if (out_axis != NULL) {
```

Comment:
This is by analogy of the others, but seems actually wrong, since the reference is borrowed.